src/control/system/database.go

```diff
@@ -493,13 +495,11 @@ func (db *Database) AddMember(newMember *Member) error {
 	db.Lock()
 	defer db.Unlock()
 
+	db.log.Debugf("newMember: %+v", newMember)
 	mu := &memberUpdate{Member: newMember}
 	if cur, err := db.FindMemberByUUID(newMember.UUID); err == nil {
-		if !cur.Rank.Equals(newMember.Rank) {
-			return errors.Errorf("re-joining server %s has different rank (%d != %d)",
-				newMember.UUID, newMember.Rank, cur.Rank)
-		}
-		return db.submitMemberUpdate(raftOpUpdateMember, mu)
+		db.log.Debugf("curMember: %+v", cur)
```

Comment:
do we want to keep this debug?

Reply:
Nope, stray debugs from hacking. Thanks.